projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/NormalSTIRFuturesExpStrikeProvider.java

```diff
@@ -17,10 +17,9 @@
 import com.opengamma.util.tuple.DoublesPair;
 
 /**
- * Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike/delay dependent.
- * The "delay" is the time between expiration of the option and last trading date of the underlying futures.
+ * Implementation of a provider of normal volatility (Bachelier model) smile for options on STIR futures. The volatility is time to expiration/strike dependent.
  */
-public class NormalSTIRFuturesSmileProvider implements NormalSTIRFuturesSmileProviderInterface {
+public class NormalSTIRFuturesExpStrikeProvider implements NormalSTIRFuturesProviderInterface {
 
   /**
    * The multicurve provider.
```

Comment:
Line 29: Java doc. Should be "Normal surface" instead of "Black cube". Remove the old TODO.